anonymizer/src/test/java/gov/va/api/health/bulkfhir/anonymizer/ClassPathResourceBasedNamesTest.java

```diff
@@ -0,0 +1,35 @@
+package gov.va.api.health.bulkfhir.anonymizer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+public class ClassPathResourceBasedNamesTest {
+
+  private List<String> testNames = List.of("Washington", "Adams", "Jefferson", "Monroe", "Madison");
+
+  @Test
+  void getName() {
```

Comment:
Change not required, but something to think about in the future... name tests with their expectation, e.g. `getNameMapsNumbersRegardlessOfListSize()` or something like that.

Reply:
:+1: